Makefile

```diff
@@ -118,15 +120,21 @@ EXCLUDE_STD := --exclude libtock_unittest --exclude print_sizes \
                --exclude runner --exclude syscalls_tests \
                --exclude libtock_build_scripts
 
-.PHONY: test
-test: examples
-	cargo test $(EXCLUDE_RUNTIME) --workspace
-	LIBTOCK_PLATFORM=nrf52 cargo fmt --all -- --check
+.PHONY: clippy
+clippy:
 	cargo clippy --all-targets $(EXCLUDE_RUNTIME) --workspace
 	LIBTOCK_PLATFORM=nrf52 cargo clippy $(EXCLUDE_STD) \
 		--target=thumbv7em-none-eabi --workspace
 	LIBTOCK_PLATFORM=hifive1 cargo clippy $(EXCLUDE_STD) \
 		--target=riscv32imac-unknown-none-elf --workspace
+
+.PHONY: format
+format:
+	LIBTOCK_PLATFORM=nrf52 cargo fmt --all -- --check
```

Comment:
The `clippy` and `format` actions need to depend on `toolchain`. In retrospect, `test` should explicitly depend on `toolchain` -- it currently transitively depends on it through `examples`.